velox/functions/prestosql/aggregates/AggregateNames.h

```diff
@@ -54,6 +54,13 @@ const char* const kMultiMapAgg = "multimap_agg";
 const char* const kReduceAgg = "reduce_agg";
 const char* const kRegrIntercept = "regr_intercept";
 const char* const kRegrSlop = "regr_slope";
+const char* const kRegrCount = "regr_count";
```

Comment:
@8dukongjian Would you add these function to documentation?

Reply:
done